imports/plugins/core/accounts/server/no-meteor/mutations/setAccountProfileCurrency.js

```diff
@@ -0,0 +1,51 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+const inputSchema = new SimpleSchema({
+  currencyCode: String,
+  accountId: String
```

Comment:
For this one, the GraphQL has optional `accountId` in the input object, so to avoid breaking changes to that, probably best to make it optional here, too, and keep the "default to viewer" behavior.